src/parsers/psmt2_to_alt_ergo.ml

```diff
@@ -349,6 +354,23 @@ module Translate = struct
       List.map2 translate_datatype_decl sort_dec datatype_dec
     with Invalid_argument _ -> assert false
 
+  let rec list_init acc n f =
+    if n > 0 then
+      list_init (f :: acc) (n - 1) f
+    else acc
+
+  let translate_push n pos =
+    try let n = int_of_string n in
```

Comment:
provide directly `n` of type int to `translate_xxx` to be able to make this function recursive and inline `list_init`.

Reply:
Since i don't lift `push` and `pop` commands anymore i directly use `mk_push pos n`.